### DISCUSSION TITLE
Accessing Specific Page Metadata Using Dataview in MetadataMenu

### DISCUSSION THREAD

**Discussion by Z4rko963** [mdelobelle/metadatamenu#645] [General]
Hello,
I’m working with MetadataMenu and Dataview in Obsidian, and I’m trying to set up a multifile field that only shows links to pages listed under pet_identification from the current page's metadata.My current approach is using the query dv.pages("${currentPagePath}").map(p => p.pet_identification), but this does not seem to work within the context of MetadataMenu. Could you guide me on the correct syntax or method to access the metadata of the current page specifically when using Dataview within MetadataMenu?Any examples or documentation pointers would be greatly appreciated.Thank you for your assistance!

![Screenshot_20240322-222131](https://github.com/mdelobelle/metadatamenu/assets/146877893/da43b901-2dca-44a1-a61a-26b113b8a829)

**Comment by mdelobelle** [comment]
can you try `current.pet_identification`?

**Comment by Z4rko963** [comment]
Hello @mdelobelle ,I wanted to extend my sincere thanks for your prompt and precise response. Your suggestion to use current.pet_identification worked perfectly. I really appreciate the support and the assistance you provide to the community. Thank you for your dedication and hard work on this project!Best regards, Emiliano zuñiga
